### ISSUE TITLE
pluginsFile config does not respect path, forces a prefixed CWD

### ISSUE THREAD

**Issue by MichaelHindley** [cypress-io/cypress#1837]
### Is this a Feature or Bug?
Bug

### Current behavior:

We have an internal framework that our other apps use for general things, where Cypress is one of them.
Our config is placed in this framework so that the consuming apps only need to write specs.
I noticed that the `pluginsFile` config does not behave like the other path based configs.

In the image below, the `supportFile` correctly points to `/frameworkpath/../support/command.js`, however the `pluginsFile`, written in the exact same manner, points to `/consumingAppPath/frameworkPath/../support/command.js` which does not exist. It seems like the path is prefixed with `process.cwd()` no matter what.

![image](https://user-images.githubusercontent.com/1016639/40772489-5ede4ed8-64c0-11e8-98a8-83d0b192ffd9.png)

### Desired behavior:
supportsFile config should be configurable to point to a supportsFile without forcing a `process.cwd()` prefix, making it impossible to have general plugin config when using a shared framework for frontend applications.

### Steps to reproduce:
See above image

### Versions
3.0.1

**Comment by chrisbreiding** [comment]
Can you describe how you're using the posted code to configure Cypress? Does it write a `cypress.json` with those values or does it use ENV variables?

**Comment by MichaelHindley** [comment]
@chrisbreiding it's used in this manner:

```js
cypress.open({
    config: cypressFolderConfig(testDirectory),
  })
``` 

where `testDirectory` is an absolute path to a ./test/ folder in the consuming application and the above code is called in the consuming apps package.json

Neither a cypress.json or .env files are used, this is exposed as a CLI.

**Comment by chrisbreiding** [comment]
Thanks for the info. I've verified this as a bug. Any absolute path won't work for the `pluginsFile` because, as you pointed out, it's prefixed with the project root path.

Should be a simple fix we can get out in the next patch release.

**Comment by brian-mann** [comment]
Released in [`3.0.2`](https://docs.cypress.io/guides/references/changelog.html#3-0-2).
